### DISCUSSION TITLE
Issue with Trim: Corrupt JPEG data?

### DISCUSSION THREAD

**Discussion by nmlizard** [ImageMagick/ImageMagick6#122] [Q&A]
Hi, 

I am having issues with trying to apply Trim to a batch scanned images of old photos. I am getting the following error on every file: 

"mogrify: Corrupt JPEG data: 1 extraneous bytes before marker 0xd9 'SCAN0380.jpg' @ warning/jpeg/c/JPEGWarningHandler/402."

The images come from an SD card (from a little FlipPal mobile scanner I use only for old family photos), so that might be a source of error, but no program has any problem opening the files.

Unfortunately, I have been unsuccessful batch-trimming these files on any program, and there are hundreds of them. Any suggestions much appreciated.

Thanks.

**Comment by dlemstra** [comment]
This is a warning instead of an error. If you add `-quiet` to the start of your command line this warning will be ignored.

**Comment by nmlizard** [comment]
OK, thank you. But the pics are not actually getting trimmed. I will post separately on that issue.
